conf/flight_plans/lagonisi_beach_survey.xml

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE flight_plan SYSTEM "flight_plan.dtd">
+
+<flight_plan alt="200." ground_alt="42" lat0="37.7776225" lon0="23.9184623" max_dist_from_home="4000" name="lagonisi_beach_survey" qfu="180" security_height="100.">
+  <header>
+#include "subsystems/ins/ins_alt_float.h"
+#include "subsystems/datalink/datalink.h"
+#include "generated/airframe.h"
+#if defined(COMMAND_FLAPS)
+#define Flaps(x) ({ (*ap_state).commands[COMMAND_FLAPS] = (int16_t)((float)MAX_PPRZ * (float)x); 0; })
+#else
+#define Flaps(x)    {0;}
+#endif
+#define RcChannel(x) ((*fbw_state).channels[RADIO_ ## x])
+</header>
+  <waypoints>
+    <waypoint height="0" name="HOME" x="14.2" y="3.2"/>
+    <waypoint height="0" lat="37.777399" lon="23.918696" name="TD"/>
+    <waypoint height="0" name="CAM_POINT" x="11.6" y="34.3"/>
+    <waypoint height="50" name="STANDBY" x="23.3" y="-67.7"/>
+    <waypoint height="200" name="RIGHT_HERE" x="138.7" y="-28.0"/>
+    <waypoint height="30" name="AF" x="36.5" y="-265.3"/>
+    <waypoint height="30" name="BASELEG" x="101.5" y="246.0"/>
+    <waypoint height="200" name="MOB" x="137.3" y="3.5"/>
+    <waypoint height="30" lat="37.774872" lon="23.918757" name="AF_SOUTH"/>
+    <waypoint height="30" lat="37.779789" lon="23.918472" name="AF_NORTH"/>
+    <waypoint height="30" name="RELEASE" x="78.7" y="-22.2"/>
+    <waypoint alt="200" lat="37.772633" lon="23.906762" name="WP1"/>
+    <waypoint alt="200" lat="37.770300" lon="23.899190" name="WP2"/>
+    <waypoint alt="200" lat="37.774123" lon="23.895975" name="WP3"/>
+    <waypoint alt="200" lat="37.771894" lon="23.885380" name="WP4"/>
+    <waypoint alt="200" lat="37.774628" lon="23.882086" name="WP5"/>
+    <waypoint alt="200" lat="37.783286" lon="23.884851" name="WP6"/>
+    <waypoint alt="200" lat="37.786916" lon="23.880351" name="TARGET"/>
+  </waypoints>
+  <sectors/>
+  <variables/>
+  <exceptions>
+    <exception cond="(RcChannel(YAW) > MAX_PPRZ/2)&&(autopilot.mode == AP_MODE_AUTO2) && (GetPosAlt() > GetAltRef()+20)" deroute="Target"/>
```

Comment:
there are `lt` and `gt` options as well as `and` keywords, but since it is your flightplan anyhow all fine. Interesting way to get what you want to do, nice example.

Reply:
It was like this in 2012 when i first started with paparazzi and i gues i got used to it due to its resemblance with C.
I will try the new way tomorrow.

Reply:
there are lt and gt options as well 
Any link to read about this?

Reply:
Sorry, not documented yet :disappointed: 
You can replace some operators with @AND @OR @LT (less than) @GT (greater than) ...
The complete list is here: https://github.com/paparazzi/paparazzi/blob/master/sw/lib/ocaml/expr_lexer.mll#L27

Reply:
Good to know!
Thank you again and of course OpenUAS.